vcr/vcr.go

```diff
@@ -406,7 +436,7 @@ func (c *vcr) find(ID ssi.URI) (vc.VerifiableCredential, error) {
 	q := leia.New(qp)
 
 	for _, t := range c.registry.Concepts() {
-		docs, err := c.store.Collection(t.CredentialType).Find(q)
+		docs, err := c.store.Collection(t.CredentialType).Find(context.TODO(), q)
```

Comment:
TODO context?

Reply:
Yes, to start a discussion, do we want unlimited time for a find by ID, or do we want to limit it tox seconds?

Reply:
if we expect calls to be returned within X seconds (e.g. 1s), I'd limit it at that.

Would be odd though, if it were to take a very long time? Would indicate a bug in storage, I'd say.

Reply:
Could be, then it would be best to fail fast.

Reply:
The limit is arbitrary, and 1s feels awfully long waiting for a fast memory-mapped store to find something. On the other side, if we hardcode it here and it occurs in a production scenario (and doesn't auto-recover), it's really blocking until we release a patched version, which is also isn't very nice..

Reply:
APIs also have a timeout, so the behaviour would be sort of the same.